tracker/tracker-vuex/src/index.ts

```diff
@@ -22,25 +46,26 @@ export default function(opts: Partial<Options> = {}) {
     }
     const encoder = new Encoder(sha1, 50);
     return store => {
-      store.subscribe((mutation, state) => {
-        if (options.filter(mutation, state)) {
+      // Vuex
+      if (store.subscribe) {
+        store.subscribe((mutation, state) => {
+          processMutationAndState(app, options, encoder, mutation, state);
+        });
+      }
+      // Pinia
+      if (store.$onAction) {
+        store.$onAction(({ name, store, args }) => {
           try {
-            const { type } = mutation;
-            if (typeof type === 'string' && type) {
-              app.send(Messages.StateAction(type));
-            }
-            const _mutation = encoder.encode(
-              options.mutationTransformer(mutation)
-            );
-            const _state = encoder.encode(options.transformer(state));
-            const _table = encoder.commit();
-            for (let key in _table) app.send(Messages.OTable(key, _table[key]));
-            app.send(Messages.Vuex(_mutation, _state));
-          } catch {
-            encoder.clear();
-          }
-        }
-      });
+            let state = {};
+            state[store.$id] = store.$state;
```

Comment:
That is a good idea. Right now it is able to capture only one store, while there can be multiple even in pure `vuex`, as far as I know. We should add this option to an original `subscribe` as well.
An additional argument `storeName: string` might be used in a wrapper function  (line 24/48) with fallback to some  default `StoreN` counter or `store.$id` in case of Pinia.

(this is not a part of changes request, just thinking out loud)